server/V2/controllers/EntryController.js

```diff
@@ -74,4 +74,28 @@ export default class Entry {
   static async specifiEntry(req, res) {
     return Response.successResponse(res, 200, 'Specific entry', req.fetchedEntry);
   }
+
+  /**
+ *
+ * @description handles delete entry
+ *
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} deleted entry details
+ *
+ */
+  static async deleteEntry(req, res) {
+    const { userid, entryid } = req.fetchedEntry;
+    const deleteEntryValues = [userid, entryid];
+    const deleteEntry = `
+    DELETE FROM entries WHERE userid = $1 AND entryid = $2 `;
+
+    try {
+      await db.pool.query(deleteEntry, deleteEntryValues);
+
+      return Response.successResponse(res, 200, 'Entry successfully deleted', req.fetchedEntry);
```

Comment:
You do not have to return the entry that has been deleted, just the status and message is fine

Reply:
I just wanted to prove the user that he delete a right entry. but that's a good idea too to just return message of confirmation and status.

Reply:
Yes it's standard to return only status and message since the data won't be useful to the frontend engineer

Reply:
sure!